server/lib/bitbucket.js

```diff
@@ -179,13 +179,16 @@ const getRulesTree = (params) =>
 const getConnectionTreeByPath = (params, filePath) =>
   new Promise((resolve, reject) => {
     try {
-      bitbucket().get(`repositories/{username}/{repo_slug}/src/{revision}/${filePath}`, params, (err, res) => {
+      bitbucket().get(`repositories/{username}/{repo_slug}/src/{revision}/${filePath}?pagelen=100`, params, (err, res) => {
         if (err) {
           return reject(err);
         } else if (!res) {
           return resolve([]);
+        } else if (!res || !res.values) {
```

Comment:
Can the 2 conditions `!res || !res.values` and `!res` be merged into a single `else if`?